### ISSUE TITLE
Scss bracket formatting

### ISSUE THREAD

**Issue by ghost** [ngant-ictu/Front-End-Engineers-Test#3]
https://github.com/ngant-ictu/Front-End-Engineers-Test/blob/fafb21cf274890813708649bdce28ac0bab598c7/Step1/scss/template.scss#L19

This bracket formatting looks like java class code. It improves readability if you follow this format for your css rule brackets:

```
.classname {
...
}
```

Please update your scss files to follow this standard convention

**Comment by grace-nguyen** [comment]
I changed.
![class](https://user-images.githubusercontent.com/13827300/39280737-2ff7765e-492b-11e8-85b4-ddbce81022f9.png)

**Comment by ghost** [comment]
@ngant-ictu Was this issue fixed in all files?

I still see instances like this:

```
.class {
  color: green; }
```

Also, you should have a space between selectors:

```
.some class {
...
}

.some-other-class {
...
}
```

NOT:

```
.some class {
...
}
.some-other-class {
...
}
```

**Comment by ngaredweb** [comment]
Yes. I fixed in all files and uploaded them to github. You can check them, i only show demo for you.

**Comment by ngaredweb** [comment]
I fixed in all files.
